src/api/models/gfcr.py

```diff
@@ -0,0 +1,355 @@
+    f3_5d = models.PositiveSmallIntegerField(
+        verbose_name="Number of people engaged in coral restoration [Indigenous]",
+        null=True,
+        blank=True,
+    )
+    f3_6 = models.PositiveSmallIntegerField(
+        verbose_name="Number of response plans (incl. financial mechanisms, eg., insurance) in place to support coral restoration after severe shocks (e.g,. storms, bleaching)",
+        default=0,
+    )
+    f4_1 = models.DecimalField(
+        max_digits=4, decimal_places=1, verbose_name="Average live hard coral cover (%)", default=0
+    )
+    f4_2 = models.DecimalField(
+        max_digits=4, decimal_places=1, verbose_name="Average macroalgae (%)", default=0
+    )
+    f4_3 = models.DecimalField(
+        max_digits=5, decimal_places=1, verbose_name="Average reef fish biomass (kg/ha)", default=0
+    )
+    f5_1 = models.PositiveSmallIntegerField(
+        verbose_name="Number of local communities engaged in meaningful participation and co-development",
+        default=0,
+    )
+    f5_2 = models.PositiveSmallIntegerField(
+        verbose_name="Number of local organizations engaged in meaningful participation and co-development",
+        default=0,
+    )
+    f5_3 = models.PositiveSmallIntegerField(
+        verbose_name="Number of local scientific/research partners involved in strengthening capacity for participation and co-development (e.g., national universities, regional science organizations)",
+        default=0,
+    )
+    f5_4a = models.PositiveSmallIntegerField(
+        verbose_name="Number of local practitioners trained / supported in coral reef conservation (e.g. community rangers) [men]",
+        default=0,
+    )
+    f5_4b = models.PositiveSmallIntegerField(
+        verbose_name="Number of local practitioners trained / supported in coral reef conservation (e.g. community rangers) [women]",
+        default=0,
+    )
+    f5_4c = models.PositiveSmallIntegerField(
+        verbose_name="Number of local practitioners trained / supported in coral reef conservation (e.g. community rangers) [youth]",
+        null=True,
+        blank=True,
+    )
+    f5_4d = models.PositiveSmallIntegerField(
+        verbose_name="Number of local practitioners trained / supported in coral reef conservation (e.g. community rangers) [Indigenous]",
+        null=True,
+        blank=True,
+    )
+    f5_5 = models.PositiveSmallIntegerField(
+        verbose_name="Number of agreements with local authorities or fishing cooperatives to manage marine resources (e.g., LMMAs, MPAs, OECMs)",
+        default=0,
+    )
+    f5_6 = models.PositiveSmallIntegerField(
+        verbose_name="Number of national policies linked to GFCR engagement, (e.g., NBSAPs, blue economy policies, national MPA declarations)",
+        default=0,
+    )
+    f6_1a = models.PositiveSmallIntegerField(
+        verbose_name="Number of direct jobs created (disaggregated by gender, age, disability, Indigenous peoples, small-scale producers) [men]",
```

Comment:
Since we're no longer doing all these, this should read `Number of direct jobs created (disaggregated by gender, age, Indigenous peoples)` -- and same for other fields with same disaggregations.